src/components/navbar/Navigation.jsx

```diff
@@ -32,6 +32,9 @@ const Navigation = () => {
               <Nav.Link href="/batchjob" style={{ color: "#fff" }}>
                 Batch Job
               </Nav.Link>
+              <Nav.Link href="/privacypolicypage" style={{ color: "#fff" }}>
+                Privacy Policy
+              </Nav.Link>
```

Comment:
Would prefer not to have this in the navbar, since it's not something that's commonly used. Can we add a footer? Something similar to what google.com has maybe?(with just Privacy)
![image](https://user-images.githubusercontent.com/53201250/133704261-396cddc9-fa5d-4aab-bde0-fdaaed18d588.png)

Reply:
I created a card for it, will try to implement it before our next team meeting.